test/_init_tests.js

```diff
@@ -402,7 +402,8 @@ function beforeTest() {
   activateChunkingForTesting();
   window.__AMP_MODE = undefined;
   window.context = undefined;
-  window.AMP_CONFIG = {};
+  // eslint-disable-next-line no-undef
+  window.AMP_CONFIG = require('../build-system/global-configs/prod-config.json');
```

Comment:
so i know this is the right thing to do, but thinking long term this would add variation to tests correct? (as in the default state could change). i guess this can't really be helped

Reply:
do you mean that there is some level of randomness (literally Math.random()) and ideally tests would not depend on any randomness whatsoever?

Reply:
more that changing a config the moment they flip a flag will force having to change the tests, but that actually seems like the desired effect.